modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -0,0 +1,115 @@
+= Best Practices to Build Mule Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As you build your applications, consider the following best practices for creating a clean, organized, and sustainable project. These practices include guidelines on using Anypoint Studio, avoiding certain building app issues, and specific settings for using Mule runtime engine (Mule) components and connectors.
+
+== Anypoint Studio Best Practices
+If you use Anypoint Studio to build Mule apps, consider the following practices:
+
+* Update to the latest version of Anypoint Studio.
+* Use the recommended Java version (OpenJDK) for the specific Studio version.
+* Include the Mule Maven plugin for automated deployment (executed by CI/CD processes).
+
+=== Define a Project Structure
+To help maintain a clean, structured, and readable project in Anypoint Studio, consider the following recommendations:
+
+* Create a separate XML for global elements, for example configuration elements.
+* Create a separate XML for each use case or resource implementation.
+* Create separate XML files for common structures or logic.
+* Create different packages for resources such as DataWeave, WSDLs, examples, and so on.
+
+==== Add Indentation and Formatting
+
+As you build your app project in Anypoint Studio, consider the following points:
+
+* Add indentation and format to all your XML (Mule XMLs, `pom.xml`, `log4j2.xml`, etc) files before committing to the source code repository.
+* Define a line width in your Anypoint Studio XML editor preferences, for example 140.d.
+* Define a naming convention.
+
+==== Manage Properties per Environment
```

Comment:
Is this title misleading? Aren't we saying that "per Environment" is NOT the way to manage properties, and that 
Manage Environment Properties Using Anypoint Runtime Manager is the point of the topic?